ThemeDecorator/I18nFontDecorator.js

```diff
@@ -1,47 +1,25 @@
 import hoc from '@enact/core/hoc';
 import ilib from '@enact/i18n';
-import {Component} from 'react';
+import {useEffect} from 'react';
 import PropTypes from 'prop-types';
 
 import {fontOverrideGenerator} from './fontGenerator';
 
 const I18nFontDecorator = hoc((config, Wrapped) => {
-	return class I18nDecorator extends Component {
-		static displayName = 'I18nFontDecorator';
+	const I18nDecorator = (props) => {
+		useEffect(() => {
+			fontOverrideGenerator(props.locale || ilib.getLocale());
+		}, [props.locale]);
```

Comment:
this useEffect() is not called whenever the locale changes in storybook, while in the previous implementation, componentDidUpdate() was called .
Please fix

Reply:
Done